Create benchmarking.md

DataBoyTX opened the pull request:
detail the benchmarking process to accurately size a system for various use cases

lmeyerov reviewed (approved):
Thanks!

Let's land this, and separately, add forward/backwards in the other capacity/deploy planning links to point to this one as well?

aucahuasi commented:
> cc @aucahuasi if anything you recommend to add, e.g., any pointers to any perf measurement usage guides

We can mention that telemetry traces are useful for Step 5: Analyze Results:
```
Use Telemetry Traces: Utilize telemetry traces to gain deeper insights into system behavior and performance bottlenecks. Graphistry supports OpenTelemetry, enabling comprehensive performance monitoring and analysis.`
```

Also, we can hint the user about testing if they have GPUDirect Storage support, something like:
```
Additional Step: Testing GPUDirect Storage Support
Check if your system supports NVIDIA GPUDirect Storage by verifying GPU and RDMA-capable adapter compatibility, ensuring drivers and software are installed, and validating GDS with benchmarking tools.  Graphistry supports RAPIDS cuDF for ETL processes.
```

DataBoyTX commented:
@lmeyerov - added two new files and updated per the comments above.  Added links from each to the other, and added the system-metrics-to-csv.sh bash script.   Please review once more.
